src/components/Quotes/Quotes.js

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from 'react';
+
+function Quotes() {
+  const [quote, setQuote] = useState('');
+  const [isLoading, setIsLoading] = useState(false);
+  const [hasError, setHasError] = useState(false);
+
+  useEffect(() => {
+    const URL = 'https://api.quotable.io/random';
+    setIsLoading(true);
+    setHasError(false);
+
+    fetch(URL)
+      .then((response) => response.json())
+      .then((data) => {
+        setQuote(data.content);
+        setIsLoading(false);
+      })
+      .catch(() => {
+        setHasError(true);
+        setIsLoading(false);
+      });
+  }, []);
```

Comment:
### [OPTIONAL]
At this moment, useEffect hook will run onMount only. For a Better User Experience, you can also pass `setIsLoading` and `setHasError` inside the dependency array so that when these methods are called then the useEffect hook will run again to update the values. 👍